lib/esbonio/esbonio/lsp/sphinx/config.py

```diff
@@ -93,6 +93,12 @@ class SphinxConfig(BaseModel):
     warning_is_error: bool = Field(False, alias="warningIsError")
     """Treat any warning as an error"""
 
+    useCustomTranslator: bool = Field(False, alias="useCustomTranslator")
```

Comment:
Assuming we don't need to be able to turn the translator off, could these two options be replaced with an `enableLivePreview` flag in the `esbonio.server` config namespace?

Reply:
On second thought. 

There might be people who don't want the sphinx build to run constantly but still retain scroll sync, so we would keep separate flags.

Reply:
Oh, true :smile: 

Could the flags be moved to [here](https://github.com/swyddfa/esbonio/blob/d83fac199988fe1212ae0128db9fb8e0fd246a88/lib/esbonio/esbonio/lsp/rst/__init__.py#L385) though? As generic `enableLivePreview` and `enableSyncScroll` flags could apply to more than just Sphinx.

Reply:
I find it a good idea to make `enableLivePreview` and `enableSyncScroll`, especially in larger projects where `enableLivePreview` might have some performance impact.